test/e2e/antreapolicy_test.go

```diff
@@ -4180,6 +4291,43 @@ func doProbe(t *testing.T, data *TestData, p *CustomProbe, protocol AntreaPolicy
 	}
 }
 
+func doEvaluation(t *testing.T, data *TestData, npEvaluation *NPEvaluation) {
+	log.Infof("running networkpolicyevaluation tests")
+	antreaController, err := data.getAntreaController()
+	if err != nil {
+		t.Errorf("Error occurred when trying to get the Antrea Controller : %v", err)
+	}
+	antctlArgs := []string{"antctl", "query", "networkpolicyevaluation"}
+	getRealizedPod := func(label Pod) string {
+		realizedPod, err := k8sUtils.GetPodByLabel(label.Namespace(), label.PodName())
+		if err != nil {
+			t.Errorf("Failed to get Pod %s: %v", label.String(), err)
+		}
+		return realizedPod.Namespace + "/" + realizedPod.Name
+	}
+	processResponse := func(stdout string) map[string]string {
+		response := make(map[string]string, 6)
+		// Split and organize table output from antctl response.
+		items := regexp.MustCompile("[^0-9A-Za-z-]+").Split(stdout, -1)
```

Comment:
There is little value in this regex which essentially just splits the string based on whitespaces.
If for the response we always expect 2 rows (header row + value), then we should have something like this:
```go
// the first row is the header row, the second row contains the actual value
rows := strings.Split(stdout, "\n")
require.Len(t, 2, rows)
headerFields := strings.Fields(rows[0])
valueFields := strings.Fields(rows[1])
require.Equal(t, len(headerFields), len(valueFields))
data := make(map[string]string)
for idx, name := range headerFields {
        data[name] = valueFields[idx]
}
```

Reply:
Sounds good, I modified the process.